### PR TITLE
0.2.0

### PR DESCRIPTION
**Pull request by johannes-wolf**
Version 0.2.0 release branch.

Yet missing:
https://github.com/johannes-wolf/cetz/milestone/2

### CODE REVIEW

File: manual.pdf
Comment (JohnMeyerhoff) [inline manual.pdf:1]: Version 0.1.1 in the title page.

Comment (johannes-wolf) [reply manual.pdf:1]: There is a huge PR #320 that update the manual/docs which fixes this.

**Comment by Andrew15-5** [comment]
@johannes-wolf, why didn't you remove the request to me from your commit? It triggered a notification for me + it looks weird (asking me in the commit, that was already committed).
https://github.com/johannes-wolf/cetz/commit/a2dc6179ef8d38c0f313f950b6de3ff4c91c71a7

**Comment by johannes-wolf** [comment]
> @johannes-wolf, why didn't you remove the request to me from your commit? It triggered a notification for me + it looks weird (asking me in the commit, that was already committed). [a2dc617](https://github.com/johannes-wolf/cetz/commit/a2dc6179ef8d38c0f313f950b6de3ff4c91c71a7)

Just squash-merged the PR tbh. I did not know GitHub notifies from commit messages.

File: gallery/karls-picture.typ
Comment (JohnMeyerhoff) [inline gallery/karls-picture.typ:1]: This looks similar but the image output is different.

File: manual.typ [hunk 1]
```diff
@@ -644,30 +367,47 @@ for (c, s, f, cont) in (
...
 
-You can also use implicit syntax of a dot separated string in the form `"name.anchor"`.
+You can also use implicit syntax of a dot separated string in the form `"name.anchor"`
+for named anchors.
 
 ```example
 line((0,0), (3,2), name: "line")
 circle("line.end", name: "circle")
-rect("line.start", "circle.left")
+rect("line.start", "circle.east")
```
Comment (JohnMeyerhoff) [inline manual.typ:387]: left -> west ( not east )
Maybe this is a wanted change, just making sure :)

**Review by JohnMeyerhoff — COMMENTED** [review]
Looks good to me! 👍 
I did not have a detailed look into the tests yet.

File: manual.typ [hunk 2]
```diff
@@ -1,183 +1,132 @@
...
-Argument types in this document are formatted in `monospace` and encased in angle brackets `<>`. Types such as `<integer>` and `<content>` are the same as Typst but additional are required:
-  / `<coordinate>`: Any coordinate system. See @coordinate-systems.
-  / `<number>`: `<integer> or <float>`
+== CeTZ Unique Argument Types
+Many CeTZ functions expect data in certain formats which we will call types. Note that these are actually made up of Typst primitives.
+  / `<coordinate>`: Any coordinate system. See coordinate-systems.
+  / `<number>`: Any of `<float>`, `<integer>` or `<length>`. 
+  / `<style>`: Named arguments (or a dictionary if used for a single argument) of style key-values.
+  / `<context>`: A CeTZ context object that holds internal state.
+  / `<vector>`: A three element array of `<float>`s
```
[the remark this answers is not in the document]
Comment (johannes-wolf) [reply manual.typ:70]: 3D

File: CHANGES.md [hunk 3]
```diff
@@ -1,15 +1,52 @@
...
+
+### Chart
+- Added `piechart` for drawing pie- and donut charts
+- Added `boxwhisker` for drawing boxwhisker charts
+
+### Decorations
+- New path decorations `zigzag`, `wave` and `coil`
 
 # 0.1.2
 CeTZ requires Typst 0.8.0.
```
[the remark this answers is not in the document]
Comment (johannes-wolf) [reply CHANGES.md:48]: I guess fully 0.10.0, but I am not sure which 0.10.0 features we use.

**Pull request merged**